ofono/include/slot.h

```diff
@@ -161,6 +161,9 @@ void ofono_slot_set_cell_info_update_interval(struct ofono_slot *s, void* tag,
 	int interval_ms);
 void ofono_slot_drop_cell_info_requests(struct ofono_slot *s, void* tag);
 
+/* Since mer/1.28+git4 */
+struct ofono_slot *ofono_slot_find_by_imsi(const char *imsi);
```

Comment:
Consider using [this API](https://github.com/sailfishos/ofono/pull/27) for querying IMEI (serial) from the modem.

Reply:
I cannot really understand why getting IMEI from the slot manager is in any way worse based on IMSI as the connection is unique on device level. But whaterver, dropped.

Reply:
And actually, I don't quite understand why would AML plugin need IMEI in the first place.

The AML service would have to fetch the list of calls anyway, to find the emergency call, it would know the associated `org.ofono.Modem` path, IMEI can be queried from there.

Reply:
Because IMEI is part of AML protocol, it is included in the messages sent. It really wouldn't need to fetch the list of calls as it is initated during a call only with a specific ID. As said, the id that is returned by the simple API extension for plugin does return the ID of the call that will be established as soon as plugin does its filtering. It is a simple solution reducing the need for yet another IPC call in the process. Simple things are more maintainable, getting the info from Ofono, that starts the AML service anyways, seems quite unnecessary as all of that info can be given as a arguments to begin with. At startup. No fuss, less processing, less communication, the better.